src/net/client/mod.rs

```diff
@@ -0,0 +1,21 @@
+//! DNS transport protocols
+#![cfg(feature = "net")]
+#![cfg_attr(docsrs, doc(cfg(feature = "net")))]
+
+//! # Example with various transport connections
+//! ```
+#![doc = include_str!("../../../examples/client-transports.rs")]
```

Comment:
The presence of this line causes `cargp test --features net` to hang. It also doesn't add anything useful to the rendered docs when `cargo doc --features net` is run.

Reply:
I don't understand this comment. For me it shows as part of the domain::net::client documentation. Indeed with cargo doc --features net

Reply:
> I don't understand this comment. For me it shows as part of the domain::net::client documentation. Indeed with cargo doc --features net

This is what I see:

<img width="1211" alt="afbeelding" src="https://github.com/NLnetLabs/domain/assets/3304436/402e27fa-944b-433d-b0bc-09d24948fe90">

What is the use of that example heading and an empty block below it?

Reply:
I just did a new checkout from github and I do see the example. I have no clue what goes wrong. Maybe move on reviewing the code? Finding issues in cargo doc doesn't seem productive at the moment